docs/tutorials/protect_against_regen.rst

```diff
@@ -0,0 +1,47 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _protect_against_regen:
+
+*************************************************************************
+Configuring Your Module to Protect Against Regeneration Voltage Spikes
+*************************************************************************
+
+What are Regenerative Voltage Spikes
+=======================================
+Regeneration voltage occurs when the commanded module voltage is less than the module's back-EMF voltage. When this occurs, the motor's kinetic 
+energy is converted directly to electrical energy, and produces a current flowing out of the motor and into the power source.
+
+How Can Regeneration Affect Your Module
+=========================================
+If the module's power source can absorb the regenerated current without allowing the voltage to spike, regeneration protection isn't needed. This is often true with a battery as the power source.
+When connected to a less than constant voltage source, like a benchtop power supply, the overall system voltage will increase during regeneration. 
```

Comment:
I'm not quite sure I understand the "less than constant" bit of this. A battery isn't a constant voltage source either, its voltage is dipping and changing all the time based on the current you draw out of it and is it discharges. Both power supplies and batteries aren't really constant, though they do have different behaviors. I would probably change this to "When connected to a power source that cannot absorb this regenerated current, which is generally the case with benchtop power supplies, the output voltage of the source will often increase during regeneration."

Reply:
Hrm. Ok, I'm fine with that. This was just stolen directly from Mike's suggestion to change what I had initially.